specification/network/resource-manager/Microsoft.Network/preview/2024-10-01-preview/networkManagerBillingMeter.json

```diff
@@ -0,0 +1,200 @@
+        "x-ms-examples": {
+          "NetworkManagerBillingMeterGet": {
+            "$ref": "./examples/NetworkManagerBillingMeterGet.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "NetworkManagerBillingMeter"
+        ],
+        "operationId": "NetworkManagerBillingMeter_Patch",
+        "description": "Patch NetworkManagerBillingMeter.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PatchObject"
+            },
+            "description": "Parameters supplied to specify Billing meter the network manager uses. Note - only VnetLevel is supported for Update operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated - Returns information about the NetworkManager BillingMeter.",
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerBillingMeter"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkManagesPatch": {
+            "$ref": "./examples/NetworkManagerBillingMeterPatch.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkManagerBillingMeter": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkManagerBillingMeterProperties",
+          "description": "The NetworkManagerBillingMeter properties"
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "./network.json#/definitions/SystemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ChildResource"
+        }
+      ],
+      "description": "The Managed Network resource"
+    },
+    "NetworkManagerBillingMeterProperties": {
+      "properties": {
+        "networkManagerId": {
+          "type": "string",
+          "description": "Network Manager Id."
+        },
+        "billingMeterType": {
+          "type": "string",
+          "$ref": "#/definitions/BillingMeterType"
+        }
+      }
+    },
+    "BillingMeterType": {
+      "type": "string",
+      "enum": [
+        "SubscriptionLevel",
+        "VnetLevel"
+      ],
+      "x-ms-enum": {
+        "name": "BillingMeterType",
+        "modelAsString": true
+      },
+      "description": "Billing Meter Type."
```

Comment:
is there a default type when the network manager is created? If so, please describe it in the description